packages/cxl-ui/src/components/cxl-marketing-nav.js

```diff
@@ -197,7 +197,6 @@ export class CXLMarketingNavElement extends LitElement {
      * @since 2024.02.14
      * @see https://app.clickup.com/t/86azbhf2a
      */
-    return '';
```

Comment:
Comment above is not needed when `return` is removed

Reply:
Removed comment.